test/stack.test.ts

```diff
@@ -0,0 +1,18 @@
+import { expect as expectCDK, matchTemplate, MatchStyle } from '@aws-cdk/assert';
+import * as cdk from '@aws-cdk/core';
+import * as Stack from '../lib/emoji-stack';
+
+test('Empty Stack', () => {
+  const app = new cdk.App();
+  // WHEN
+  const stack = new Stack.EmojiStack(app, 'MyTestStack');
+  // THEN
+  expectCDK(stack).to(
+    matchTemplate(
+      {
+        Resources: {},
+      },
+      MatchStyle.EXACT
+    )
+  );
+});
```

Comment:
Default test can be removed if it's not gonna test anything.

Reply:
Done, although I've had to manually install the `debug` dependency because `axios` indirectly depends on it but it's not installed as sub-dependency or marked as peer dependency 🤷